Resources/Prototypes/Entities/Constructible/Furniture/seats.yml

```
@@ -22,6 +22,7 @@
     sprite: Constructible/Misc/furniture.rsi
   - type: Strap
     position: Stand
+    buckleOffset: "0,.15"
```

```suggestion
    buckleOffset: "0,0.15"
```
If this is valid anyway, reads a bit cleaner.